manifests/a/Amazon/AWSCLI/2.6.0.0/Amazon.AWSCLI.locale.en-US.yaml

```diff
@@ -14,6 +17,7 @@ Copyright: Copyright 2012-2020 Amazon.com, Inc.
 CopyrightUrl: https://github.com/aws/aws-cli/blob/develop/LICENSE.txt
 ShortDescription: Universal Command Line Interface for Amazon Web Services
 Description: The AWS CLI is an open source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services. With minimal configuration, you can start using all of the functionality provided by the AWS Management Console from your favorite terminal program.
+Moniker: adoptopenjdk8
```

Comment:
No

Reply:
Yeah! I know, that is why I canceled the run.